addons/hr_timesheet/models/res_config_settings.py

```diff
@@ -20,3 +20,11 @@ class ResConfigSettings(models.TransientModel):
         and widgets to help the encoding. All reporting will still be expressed in hours (default value).""")
     timesheet_min_duration = fields.Integer('Minimal duration', default=15, config_parameter='hr_timesheet.timesheet_min_duration')
     timesheet_rounding = fields.Integer('Rounding up', default=15, config_parameter='hr_timesheet.timesheet_rounding')
+    # Used to hide "timesheet_min_duration" and "timesheet_rounding" in view.
+    # If (is_encode_uom_days == True) and (module_project_timesheet_synchro == False), they are hidden.
+    is_encode_uom_days = fields.Boolean(compute='_compute_is_encode_uom_days')
+
+    @api.depends('timesheet_encode_uom_id')
+    def _compute_is_encode_uom_days(self):
+        for settings in self:
+            settings.is_encode_uom_days = settings.timesheet_encode_uom_id == self.env.ref('uom.product_uom_day')
```

Comment:
get the ref only one time, outside the for loop

Reply:
Done